File: 황창식/C_sharp/Class_practice_20210208/Class_practice_2/SumAve.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Class_practice_2
{
    class SumAve
    {
        private int sum;
        private int ave;
        private int[] arr;

        public int Sum { get => sum; set => sum = value; }
        public int Ave { get => ave; set => ave = value; }
        public int[] Arr { get => arr; set => arr = value; }

        public SumAve(){ }
        public SumAve(int[] arr)
        {
            getSumAve(arr);
        }

        public void getSumAve(int[] arr)
        {
            for (int i = 0; i < arr.Length; i++)
            {
                sum += arr[i];
            }
            ave = sum / arr.Length;

            Console.WriteLine("총점 : " + sum);
            Console.WriteLine("평균 : " + ave);
        }

        public void getSumAve()
        {
            for (int i = 0; i < arr.Length; i++)
            {
                sum += arr[i];
            }
            ave = sum / arr.Length;

            Console.WriteLine("총점 : " + sum);
            Console.WriteLine("평균 : " + ave);
        }



        /*public int getSum(int[] arr)
        {
            sum = 0;
            for (int i = 0; i < arr.Length; i++)
            {
                sum += arr[i];
            }
            return sum;
        }

        public int getAve(int[] arr, int sum)
        {
            ave = sum / arr.Length;
            return ave;
        }*/

    }
}

File: 황창식/java/class_practice/sumave/src/sumave/SumAve.java
package sumave;

public class SumAve {
	private int sum;
	private int ave;
	private int arr[];
	
	public int[] getArr() {
		return arr;
	}

	public void setArr(int[] arr) {
		this.arr = arr;
	}
	
	public SumAve(int[] arr) {
		super();
		this.arr = arr;
	}
	
	public int getSum() {
		for (int i = 0; i < arr.length; i++) {
			sum += arr[i];
		}
		return sum;
	}
	
	public int getAve() {
		ave = sum/arr.length;
		return ave;
	}
	
	public void showResult() {
		for (int i = 0; i < arr.length; i++) {
			System.out.println(arr[i]);
		}
		System.out.println("총점 : " + getSum());
		System.out.println("평균 : " + getAve());
	}
}
